<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>201</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>State / City</string>
    </property>
    <widget class="QComboBox" name="stateMenu">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>State:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="cityList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>81</y>
       <width>181</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>City:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>199</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Zip Code:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="zipCodeList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>181</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="bCategoryList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>181</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Business Category:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>591</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Businesses</string>
    </property>
    <widget class="QListWidget" name="businesses">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>571</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="bname">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Business Name:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
